osbs/api.py

```diff
@@ -280,11 +280,8 @@ def create_simple_build(self, git_uri, git_ref, git_branch, user, component, yum
         build_request.set_params(
             git_uri=git_uri,
             git_ref=git_ref,
-            git_branch=git_branch,
             user=user,
             component=component,
-            base_image=df_parser.baseimage,
-            name_label=df_parser.labels['Name'],
             registry_uri=self.build_conf.get_registry_uri(),
```

Comment:
So this looks like `create_simple_build` won't even need `df_parser` and it can be removed altogether from that method.